<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>采集设置</string>
     </attribute>
     <widget class="QComboBox" name="collection_source">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Simulation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Socket</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="collection_settings_save">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>采集源</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="count_time_option">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>计数时间</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="total_count_option">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>道计数</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="count_time_setting">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>30</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tota_count_setting">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>380</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100000000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>采集终止标准</string>
      </property>
     </widget>
     <widget class="QPushButton" name="link_source">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>连接采集源</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cut_source">
      <property name="geometry">
       <rect>
        <x>144</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>断开采集源</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="connection_icon_layout"/>
     </widget>
     <widget class="QLineEdit" name="ip_addr_input">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="port_input">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>6688</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IP地址</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>显示设置</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>量程</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="count_range">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>29</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1000000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lower_bound">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="upper_bound">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1024</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>79</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>局部谱</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>85</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>到</string>
      </property>
     </widget>
     <widget class="QPushButton" name="display_setting_save">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>470</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提交设置</string>
      </property>
     </widget>
     <widget class="QComboBox" name="plot_method">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Line</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scatter</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>曲线绘制方式</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>曲线扩展方式</string>
      </property>
     </widget>
     <widget class="QComboBox" name="curve_expansion">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Original</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Interval</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Max</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mean</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>曲线颜色</string>
      </property>
     </widget>
     <widget class="QComboBox" name="curve_color">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Red</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>是否对数显示</string>
      </property>
     </widget>
     <widget class="QComboBox" name="log_count">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>300</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>存储路径设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="set_path">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cur_path_display">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>./output/</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>当前信息</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>采集源</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="time_elapsed">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>82</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>采集时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>总计数</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="total_count">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="collection_setting_display"/>
     </widget>
     <widget class="QProgressBar" name="finished_percentage">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>采集完成进度</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="peak_area">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>312</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>峰面积</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="half_wave">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>362</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>半高宽</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="integral">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>412</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>积分</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="count_rate">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>252</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>计数率(/s)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>821</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QCommandLinkButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="pause_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停采集</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="end_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>结束采集</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="export_spectrum">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>510</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导出能谱</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
